Rewrite the README to reflect the v6 changes

rdwebdesign opened the pull request:
**Note:** This PR depends on the graphics repo PR to include the new images.

---

### What does this PR aim to accomplish?

Change the README file to reflect the v6 changes, showing the new features and removing the deprecated ones.

### How does this PR accomplish the above?

Adding new images a and changing texts where needed.

---
**By submitting this pull request, I confirm the following:**

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against.
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
6. I have checked that another pull request for this purpose does not exist.
7. I have considered, and confirmed that this submission will be valuable to others.
8. I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
9. I give this submission freely, and claim no ownership to its content.

---
- [x] I have read the above and my PR is ready for review. *Check this box to confirm*

rdwebdesign committed:
Rewrite the README to reflect the v6 changes

- Change the initial logo block:
  - simplify the logo code using a SVG logo instead of 2 PNG files;
  - the new SVG file contains `prefers-color-scheme` CSS rule to change
    text color depending on the user preferences (dark/light modes)
  - add some links: Pi-hole site, Discourse, Documetation and Donate

- Remove a broken link to Pi-hole store

- Use new images and change a few texts to show v6 features

- Remove "Audit log" and "Debug log" sections (they don't exist in v6)

Signed-off-by: RD WebDesign <github@rdwebdesign.com.br>

rdwebdesign committed:
More changes:

- Replace some more images (v6 version);
- Add text description for settings;
- Use TEX code to highlight some text labels;

Signed-off-by: RD WebDesign <github@rdwebdesign.com.br>

yubiuser commented on README.md:
```diff
@@ -1,26 +1,31 @@
+
+# Pi-hole Web Interface
+
+<img src="https://pi-hole.github.io/graphics/Screenshots/v6/v6_dashboard.png" alt="Pi-hole Web interface">
```
Depending on https://github.com/pi-hole/web/pull/3139 we might need another new image

rdwebdesign replied:
I already prepared a branch with new images, just in case #3139 is approved.

yubiuser reviewed (changes requested):
What is the browser-stack-badge for? I don't think we use that service...

__

I would remove the link to disourse on how to access the web interface.

__

We can now also access via `https`

____

Remove "forgot your password?"

___

Passwort protection - maybe add 2FA + Screenshot

rdwebdesign commented:
> What is the browser-stack-badge for?

I really don't know. 
This badge was there since a long time ago. I will remove it because we don't use this service.

rdwebdesign committed:
Implementing suggestions:

- remove browserstack badge;
- remove unnecessary texts and links;
- add a note about http/https access;

Signed-off-by: RD WebDesign <github@rdwebdesign.com.br>

rdwebdesign committed:
Change icon sizes and alignment (PayPal and Bitcoin)

Signed-off-by: RD WebDesign <github@rdwebdesign.com.br>

rdwebdesign commented:
> What is the browser-stack-badge for? I don't think we use that service...
- Done: badge removed

> I would remove the link to disourse on how to access the web interface.
- Done: Agree... link removed

> We can now also access via https
- Done: note added to make the use of http/https clear;

> Remove "forgot your password?"
- Done: text removed

> Passwort protection - maybe add 2FA + Screenshot
- I will change the "login screen" image in a future PR.

PromoFaux commented:
> This badge was there since a long time ago. I will remove it because we don't use this service.

We might not have used it recently, but we have a sponsored plan with the service, so perhaps put it back for the time being (I think it was a condition of the plan)

rdwebdesign commented:
I will wait until we decide what will happen with https://github.com/pi-hole/web/pull/3139.

yubiuser commented:
> sponsored plan with the service

Do you want to re-enable the service to check the web interface?

rdwebdesign committed:
Readd the browserstack add

Signed-off-by: RD WebDesign <github@rdwebdesign.com.br>
